lib/json_api_assert.ex

```diff
@@ -55,6 +55,35 @@ defmodule JsonApiAssert do
     do: raise ExUnit.AssertionError, "jsonapi object not found"
 
   @doc """
+  Asserts that a valid meta object exists in the payload
+
+    payload
+    |> assert_meta([:copywrite, :authors])
```

Comment:
Alright, I can open another PR to add this but I'll be away from a computer for the next 8 hours or so.  I'll make the change here as well.